spec/unit/type/service_spec.rb

```diff
@@ -103,6 +103,28 @@ def safely_load_service_type
     end
   end
 
+  # currently, the only provider that uses timeout is the windows service provider, so confine the tests to that
+  if Puppet.features.microsoft_windows?
```

Comment:
It might be better to just add the `configurable_timeout` feature like in e.g. https://github.com/puppetlabs/puppet/blob/master/spec/unit/type/user_spec.rb#L6-L17 since parameters are not supposed to be platform-specific but feature-specific, which in this case happens to be a Windows-only thing. This would remove the `if` statement.

Reply:
Ahhh, thank you! I was looking for an example like that.